### DISCUSSION TITLE
Why are hypr builds not included anymore?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [xremap/xremap#526] [Q&A] [answered]
The latest version hypr build version is 0.8.18.

**Comment by k0kubun** [comment]
Since hypr is based on wlroots, wlroots builds took over the role https://github.com/xremap/xremap/blob/master/CHANGELOG.md#v0100. Please use wlroots builds instead.

**Reply by azizsayfiddin** [reply]
It's not dependant on wlroots [anymore](https://hyprland.org/news/independentHyprland/)

**Reply by k0kubun** [reply]
Oh... I wasn't aware of that. Thank you for letting me know. I'll probably just resurrect the hypr binary then. Please consider using v0.8 in the meantime.

**Reply by k0kubun** [reply, marked as answer]
Fixed it in v0.10.2.
